topics/transcriptomics/tutorials/goenrichment/tutorial.md

```diff
@@ -0,0 +1,343 @@
+objectives:
+  - "How to perform a GO Enrichment Analysis"
+  - "How to interpret and simplify the results"
+time_estimation: "1h"
+key_points:
+  - "The goenrichment tool can be used to perform GO Enrichment analysis"
+  - "One needs to be careful when chosing the background population"
+  - "There are several methods to simplify the output of the GO Enrichment analysis"
+contributors:
+  - DanFaria
+  - dsobral
+  - joanapaulas
+---
+
+# Introduction
+{:.no_toc}
+
+When we have a large list of genes of interest (for example, a list of differentially expressed genes obtained from an RNA-Seq experiment), how do we extract biological meaning from that list?
+
+One way is to perform functional enrichment analysis. This method consists of the application of statistical tests to verify if genes of interest are more often associated to certain biological functions than what would be expected in a random set of genes. In this tutorial you will learn about enrichment analysis and how to perform it.
+
+**What is the Gene Ontology?** <br/>
+The [Gene Ontology](http://www.geneontology.org/) (GO) is a structured, controlled vocabulary and classification of gene function at the molecular and cellular level. It is divided in three separate sub-ontologies or GO types: biological process (e.g., signal transduction), molecular function (e.g., ATPase activity) and cellular component (e.g., ribosome). These sub-ontologies are structured as directed acyclic graphs (a hierarchy with multi-parenting) of GO terms.
+ 
+![](../../images/goenrichment_GOexample.png)
+> 
+**Figure 1** QuickGO - http://www.ebi.ac.uk/QuickGO
+
+**What are GO annotations?** <br/>
+Genes are associated to GO terms via GO annotations. Each gene can have multiple annotations, even of the same GO type. An important notion to take into account when using GO is that, according to the **true path rule**, a gene annotated to a term is also implicitly annotated to each ancestor of that term in the GO graph. GO annotations have evidence codes that encode the type of evidence supporting them: only a small minority of genes have experimentally verified  annotations; the large majority have annotations inferred electronically based on sequence homology or known patterns.
+
+> ### Overview
+>
+> In this tutorial, we will deal with:
+> 
+> 1. Functional Enrichment Analysis
+> 2. Methods to simplify the results
+> 3. Interpretation of the results
+> {:toc}
+>
+{: .agenda}
+
+ 
+# Functional Enrichment Analysis
+
+To perform functional enrichment analysis, we need to have:
+- A set of genes of interest (e.g., differentially expressed genes): **study set**
+- A set with all the genes to consider in the analysis: **population set** (which must contain the study set)
+- GO annotations, associating the genes in the population set to GO terms
+- The GO ontology, with the description of GO terms and their relationships
```

Comment:
I can see it in the screenshot but what about adding here, or somewhere in the text, that the ontology file needs to be in OBO or OWL format, maybe with a sentence on what the format is or a link to an explanation? e.g. http://www.geneontology.org/faq/what-obo-file-format, http://www.geneontology.org/faq/what-owl-file. And/or a link in the text to where the files come from (http://geneontology.org/page/download-ontology) could be good. Those ontology files can be used with any species can they? As maybe could be good to explicitly state that.